packages/bsky/src/views/index.ts

```diff
@@ -1050,6 +1130,16 @@ export class Views {
     let recordInfo: RecordInfo<Record<string, unknown>> | null | undefined
     if (uri.collection === ids.AppBskyFeedPost) {
       recordInfo = state.posts?.get(notif.uri)
+      // filter hidden replies
+      if (recordInfo && notif.reason === 'reply') {
+        const rootPostUri = isPostRecord(recordInfo.record)
+          ? recordInfo.record.reply?.root.uri
+          : undefined
+        const isHiddenReply = rootPostUri
+          ? this.replyIsHidden(notif.uri, rootPostUri, state)
+          : false
+        if (isHiddenReply) return
+      }
```

Comment:
Filters out hidden replies for _everyone's_ notifications. Replies to a hidden reply are not filtered out.

Reply:
Removed this, will not filter from notifications feed for now, and if we do, we'll do it at the dataplane